src/net/sf/memoranda/ui/TaskPanel.java

```diff
@@ -513,7 +519,8 @@ void newTaskB_actionPerformed(ActionEvent e) {
  			ed = null;
         long effort = Util.getMillisFromHours(dlg.effortField.getText());
 		//XXX Task newTask = CurrentProject.getTaskList().createTask(sd, ed, dlg.todoField.getText(), dlg.priorityCB.getSelectedIndex(),effort, dlg.descriptionField.getText(),parentTaskId);
-        Task newTask = CurrentProject.getTaskList().createTask(sd, ed, dlg.todoField.getText(), dlg.priorityCB.getSelectedIndex(),effort, dlg.descriptionField.getText(),(String)dlg.categoryCB.getSelectedItem(), null);
+        //My change of same line at end
+        Task newTask = CurrentProject.getTaskList().createTask(sd, ed, dlg.todoField.getText(), dlg.priorityCB.getSelectedIndex(),effort, dlg.descriptionField.getText(),(String)dlg.categoryCB.getSelectedItem(), (String)dlg.timerCB.getSelectedItem()null);
```

Comment:
wow thanks, will give it a try!

On Sat, Feb 13, 2016 at 5:25 PM, smbende1 <notifications@github.com> wrote:

> In src/net/sf/memoranda/ui/TaskPanel.java
> <https://github.com/cst316/spring16project-Team-El-Paso/pull/25#discussion_r52835785>
> :
>
> > @@ -513,7 +519,8 @@ void newTaskB_actionPerformed(ActionEvent e) {
> >   			ed = null;
> >          long effort = Util.getMillisFromHours(dlg.effortField.getText());
> >  		//XXX Task newTask = CurrentProject.getTaskList().createTask(sd, ed, dlg.todoField.getText(), dlg.priorityCB.getSelectedIndex(),effort, dlg.descriptionField.getText(),parentTaskId);
> > -        Task newTask = CurrentProject.getTaskList().createTask(sd, ed, dlg.todoField.getText(), dlg.priorityCB.getSelectedIndex(),effort, dlg.descriptionField.getText(),(String)dlg.categoryCB.getSelectedItem(), null);
> > +        //My change of same line at end
> > +        Task newTask = CurrentProject.getTaskList().createTask(sd, ed, dlg.todoField.getText(), dlg.priorityCB.getSelectedIndex(),effort, dlg.descriptionField.getText(),(String)dlg.categoryCB.getSelectedItem(), (String)dlg.timerCB.getSelectedItem()null);
>
> Looks like there is a missing comma here at the end.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cst316/spring16project-Team-El-Paso/pull/25/files#r52835785>
> .
>



-- 
AJ Kowal
(480) 258-2794